content/tutorials/_index.md

```diff
@@ -6,4 +6,6 @@ lastmod: 2021-03-19T17:03:04+01:00
 weight: 50
 cascade:
   type: tutorials
----
+  variables:
+    gatlingBundleVersion: "3.7.0"
```

Comment:
Shouldn't be the same as https://github.com/gatling/frontline-cloud-doc/blob/main/content/reference/_index.md?plain=1#L5 ?

Reply:
Better: shouldn't we share the constants at the root, instead of having different ones for each branch?